test/quantization/test_quantize.py

```diff
@@ -1295,6 +1298,25 @@ def checkNoFQModule(m):
         m = convert(m)
         checkNoFQModule(m)
 
+    def test_relu(self):
+        class M(torch.nn.Module):
+            def __init__(self):
+                super().__init__()
+                self.relu = nn.ReLU()
+
+            def forward(self, x):
+                x = self.relu(x)
+                return x
+
+        m = M().train()
+        m.qconfig = default_qconfig
+        m = prepare_qat(m)
```

Comment:
For ReLU6, would the same logic apply?

Reply:
yes, but relu6 requires some more changes, we need to merge the implementation of qunatized relu6 to the default one.